kubectl-pg/README.md

```diff
@@ -0,0 +1,53 @@
+# kubectl-pg
+
+### Installtion of kubectl pg plugin
+
+Clone the kubectl pg plugin and build from the source using ```go install``` this will generate kubectl pg plugin's executable in ```go/bin/kubectl-pg.exe```
+
+### To list all commands avaialble in kubectl pg
+
+```kubectl pg --help``` (or) ```kubectl pg```
+
+### To validate postgresql CRD creation
+
+```kubectl pg check```
+
+### To create postgresql cluster using manifest file
+
+```kubectl pg create -f acid-minimal-cluster.yaml```
+
+### To update existing cluster using manifest file
+
+```kubectl pg update -f acid-minimal-cluster.yaml```
+
+### To delete existing cluster using manifest file
+
+```kubectl pg delete -f acid-minimal-cluster.yaml```
+
+### To delete existing cluster using cluster name
+
+```kubectl pg delete acid-minimal-cluster```
+
+--namespace or -n flag to specify namespace if cluster is in other namespace.
+
+```kubectl pg delete acid-minimal-cluster -n namespace01```
+
+### To list postgres resources for the current namespace
+
+```kubectl pg list```
+
+### To list postgres resources across namespaces
+
+```kubectl pg list all```
+
+### To add-user and roles to existing cluster
+
+```kubectl pg add-user USER01 -p CREATE,UPDATE,DELETE -c acid-minimal-cluster```
+
+### To add-db and assigning owner to existing cluster
+
+```kubectl pg add-db DB01 -o OWNER01 -c acid-minimal-cluster```
+
+### To extend volume for an existing cluster
```

Comment:
Does this command extend the volume by 2Gi (`new_size == old_size + 2 Gi`) or to 2 GI (`new_size == 2 Gi`)?

Reply:
Okay let me clarify how I implemented this, I might be wrong in verbiage. Whatever ext-volume will be provided it will be the updated volume, not the old volume+provided new volume. Let me know if the expected is to extend the provided volume by adding it to the existing volume.

Reply:
that's totally fine for now